test/integration/concierge_jwtauthenticator_status_test.go

```diff
@@ -0,0 +1,297 @@
+// Copyright 2024 the Pinniped contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"go.pinniped.dev/generated/latest/apis/concierge/authentication/v1alpha1"
+	"go.pinniped.dev/test/testlib"
+)
+
+// Never run this test in parallel since deleting all federation domains is disruptive, see main_test.go.
+func TestConciergeJWTAuthenticatorStatus_Disruptive(t *testing.T) {
+	env := testlib.IntegrationEnv(t)
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
+	t.Cleanup(cancel)
+
+	tests := []struct {
```

Comment:
Aren't there more cases here that could be easily tested? Like when the CA is invalid, when the CA is a valid CA but does not match the server's, etc. This integration test does not need to test every case that the controller's unit test handles, but it could test all the cases that are easy to cause.

Reply:
Sure thing. I was intentionally going minimal with integration testing, but can definitely expand to a few more cases.